shared/aux.go

```diff
@@ -252,12 +256,67 @@ func JoinCommands(cmd, kubeconfigFlag string) string {
 	return joinedCmd
 }
 
+// GetJournalLogs returns the journal logs for a specific product
+func GetJournalLogs(product, ip string) (string, error) {
```

Comment:
sure, but i was thinking about the caller of this, whoever wants those logs, that handle in his way what to do with the logs.
that way we maintain the responsability with the consumer and also do not tie this up to a specific data type ( json,txt , etc ) and do not tie this up to a specific path file.

Reply:
Sounds good, I think the journallogs can be dumped as just plain text by default to help provide the dev for debugging purpose, this can be a separate task as well